CMakeLists.txt

```diff
@@ -656,7 +705,9 @@ install(DIRECTORY extlibs/ttmath/ttmath
 # Prepare pkg-config file
 
 set(NETWORKIT_LIBNAME networkit)
-execute_process(COMMAND python3 -c "from version import version; print(version, end='')"
+set(NETWORKIT_CMAKE_PYTHON_INTERPRETER ${Python_EXECUTABLE})
```

Comment:
Why do we use this `set()` instead of replacing all occurrences of `NETWORKIT_CMAKE_PYTHON_INTERPRETER` by `Python_EXECUTABLE`?